applications/mpsky/templates/service.yaml

```
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: "mpsky"
+  labels:
+    {{- include "mpsky.labels" . | nindent 4 }}
+  annotations:
+#    metallb.universe.tf/address-pool: sdf-services
+    loadBalancerIP: 172.24.5.245
```

This IP is being baked into the service template, and therefore will be the same for all Phalanx environments where mpsky is deployed (usdf-rsp-dev, usdf-rsp, etc.). I _think_ this may need to be different for each deployment. To do this, add a value to `values.yaml` that will get overriden for each environment (i.e., `values-usdfdev.yaml`). To handle that awkward initial deployment, you could do something like set the default loadBlancerIP to "none" and do a conditional check to use the sdf-services annotation instead, like this (I haven't checked this syntax, so try this first):

```suggestion
  annotations:
    {{- if .Values.loadBalancerIP == "none" }}
    metallb.universe.tf/address-pool: sdf-services
    {{- else }}
    loadBalancerIP: {{ .Values.loadBalancerIP | quote }}
    {{- end }}
```

Last, add a comment here that captures the discussion from Slack about how this IP address is determined.